clients/java/signalr/src/main/java/Transport.java

```diff
@@ -3,8 +3,8 @@
 
 public interface Transport {
     void start() throws InterruptedException;
```

Comment:
I think the InterruptedException here is specific to how WebSockets transport is started. Http based transports might throw different exceptions which you may need to list here. If someone comes up with a different transport (unlikely) that needs to throw yet another exception they won't be able to implement this interface. Similarly if you decide to use a different WebSockets library of Http client you may not be able to switch without breaking users.